src/pages/certificates/BrowserImport.tsx

```diff
@@ -132,6 +152,33 @@ const BrowserImport: FC = () => {
             </ul>
           </div>
         )}
+
+        {activeTab === MACOS && (
+          <div role="tabpanel" aria-label="safari macos">
+            <ul className="p-list--divided u-no-margin--bottom">
+              <li className="p-list__item">
+                The certificate must be protected by password. An empty password
+                will fail to be imported on macOS.
+              </li>
+              {downloadPfx}
+              <li className="p-list__item">
+                Start the Keychain Access app on your Mac, select the login
+                keychain.
+              </li>
+              <li className="p-list__item">
+                Drag the certificate file onto the Keychain Access app.
```

Comment:
I find this a bit confusing. Maybe instead of "the certificate file" we could explicitly say "the lxd-ui.pfx file"? So that we disambiguate between the .crt and the .pfx.